### PR TITLE
updated module import 

### PR DESCRIPTION
**Pull request by knakul853**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@ haversine
 httplib2
 -e git://github.com/pchampin/hydra-py.git#egg=hydra
```
Comment (Mec-iS) [inline requirements.txt]: We don't need the `-e` flag. Please substitute these lines by using the new approach as displayed [here](https://github.com/HTTP-APIs/hydrus/blob/develop/requirements.txt#L26)

**Comment by Mec-iS** [comment]
See my comment in the code.

The commit message is really poor. Read [here](https://chris.beams.io/posts/git-commit/) how to write a proper commit message.

**Comment by knakul853** [comment]
@Mec-iS sorry about poor comment

**Comment by Mec-iS** [comment]
You have to use `develop` as base branch. 

You cannot just go and add a new commit; to correct a previous commit message first do `git reset HEAD~` to go back one commit, apply the changes and re-commit, then push to remote using `git <your origin> <your branch> --force`. This way we will have a clean commits history.

**Comment by Mec-iS** [comment]
There is already #15 that is working on this issue. If you want to add something, please collaborate in that issue by forking and merging in the branch there.

**Comment by knakul853** [comment]
@Mec-iS How can I collaborate on that pr. can you provide me steps regarding this. sorry to disturb you.

**Comment by Mec-iS** [comment]
1. Fork [your colleagues branch from its repository](https://github.com/nekiter/hydra-flock-central-controller/tree/develop)
2. open a PR to its branch with your changes
3. once the changes are merged there, they will automatically be added to the PR (as the PR and the fork use the same branch)

**Pull request closed without merging**